src/System.Net.Http.WinHttpHandler/src/System/Net/Http/WinHttpHandler.cs

```diff
@@ -868,8 +610,7 @@ private static bool IsChunkedModeForSend(HttpRequestMessage requestMessage)
             return chunkedMode;
         }
 
-        // TODO: Refactor to smaller files. Perhaps split cookie handling into separate class.
-        private static string GetCookieHeader(Uri uri, CookieContainer cookies)
+        internal static string GetCookieHeader(Uri uri, CookieContainer cookies)
```

Comment:
Design: Instead of making this internal, why not move it to a separate Cookie handling class?